platform/ios/src/MGLPinAnnotationView.m

```diff
@@ -0,0 +1,72 @@
+#import "MGLPinAnnotationView.h"
+
+@implementation MGLPinAnnotationView
+
+- (instancetype)initWithReuseIdentifier:(nullable NSString *)reuseIdentifier {
+    self = [super initWithReuseIdentifier:reuseIdentifier];
```

Comment:
By convention, this statement is the condition of an `if` statement, and all the properties below are set inside the `if` statement. If the superclass initializer fails for some reason by returning `nil`, it may not be safe to modify any of `self`’s storage.